### ISSUE TITLE
[a11y Design Review]: Finding: a11y-defect-2: Heading Order - Veteran Support Team: Feedback Mechanism - Rate Your Experience

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#67861]
# Testing Areas

- [x] **Desktop**: No issues found.
- [x] **Mobile**: Issue found.

<hr>

## Desktop Testing

<details><summary>Toggle to view desktop testing results</summary>

- **Operating System:** Mac + Windows
- **Assistive Technology being used:** None - [Heading Order](https://accessibility.18f.gov/headings/)
- **Internet Browser:** Chrome
- **Device Type:** Desktop 
- [**Desktop Prototype Link**](https://www.figma.com/file/s9aAXodKPxFE8ffYO8vg0T/Rate-tool?type=design&node-id=70-1740&mode=design&t=zDsLFnvwUgxYEK8i-0)

### Issue(s)
None found. 
</details>

## Mobile Testing

<details><summary>Toggle to view mobile testing results</summary>

- **Operating System:** Mac + Windows
- **Assistive Technology being used:** None - [Heading Order](https://accessibility.18f.gov/headings/)
- **Internet Browser:** Chrome
- **Device Type:** Desktop (testing Mobile via Figma)
- [**Mobile Prototype Link**](https://www.figma.com/file/s9aAXodKPxFE8ffYO8vg0T/Rate-tool?type=design&node-id=87-3553&mode=design&t=zDsLFnvwUgxYEK8i-0)

### Steps to recreate

- Log into Figma
- Open prototype in Figma
- Click on mobile, under pages. 
- Locate the H4 in the left-hand panel 
- Click the heading above it that renders H2

### Issue

Your content doesn't use correct and consistent labels across links, entry points, navigation components, and `H1s`.

#### Example 1

Heading level 3 `<h3>` is displaying after a heading level 4 `<h4>`.

<img width="619" alt="Heading level 3 is displaying after a heading level 4" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/124186314/0f7c32a1-2052-4191-a168-e1233c928c3e">

This heading is labeled in the prototype as a `<h4>`.

<img width="578" alt="This heading is labeled in the prototype as a H4." src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/124186314/9137ed26-8211-4125-ac1e-01f04004b252">

This heading is labeled in the prototype as an `<h3>`.

<img width="460" alt="This heading is labeled in the prototype as an H3." src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/124186314/1976b9e1-14d3-4011-b2d4-8452a53aa60b">

#### Example 2

Heading level 4 `<h4>` is displaying after a heading level 2 `<h2>`.

<img width="638" alt="Heading level 4 is displaying after a heading level 2" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/124186314/9acab6e4-f68e-4103-89d6-cc5a054da72f">

### Issue Description

HTML should have properly nested headings. Adding an `<h4>` as the first heading underneath an `<h2>` is an example of improper nesting. The WebAIM WAVE or headingsMap browser plugins provide visual representations of heading nesting.

- **Impact Areas**: A11y, Content and IA
- **Experience Standards:** Issue 2, Category 8
- **Severity Rubric:** a11y-defect-2 
    - **Severity**: Depends
    - **Timing**: Serious, should be fixed in 1-2 sprints post-launch
- [WCAG Level: AA (resource)](https://www.w3.org/WAI/WCAG21/Understanding/headings-and-labels)

#### Heading elements should be written in a logical sequence.

The order of heading elements should descend, based on the “depth” of the content. For example, a h4 element should not appear on a page before the first h3 element declaration. A tool such as headingsMap can help you evaluate this.

#### Don't skip Heading Levels

For example, don't jump from a h2 to a h4, skipping a h3 element. If heading levels are being skipped for a specific visual treatment, use CSS classes instead.

### Additional Information

Heading elements (h1, h2, h3, etc.) help break up the content of the page into related “chunks” of information. They are incredibly important for helping people who use assistive technology to understand the meaning of a page or view.

Heading elements construct a document outline, and should not be used for purely visual design.

</details>

**Comment by sara-amanda** [comment]
## CAIA Design Review Tickets
- [Primary Design Review Ticket 67857](https://github.com/department-of-veterans-affairs/va.gov-team/issues/67857#issuecomment-1769361642) `CLOSED AND SHARED ON 10/18/2023`
- Closing this ticket `67861` as well, now that findings were shared.

**cc:** @UXCamG @naomimarcussen
